metrics/main.py

```diff
@@ -462,31 +468,36 @@ def run_analysis(
 # Flags for Vision OCP study
 GENERATE_HEATMAPS = False
```

Comment:
please revert these flag changes and create a new set of flags for your own study